build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+pipenv lock -r --pre > requirements.txt
```

Comment:
Is this going to generate untracked changes when it runs?

Reply:
Swapped `requirements.txt` for `pipenv` entirely, per our conversation.
f4cf624